tests/test_kokkos.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
```

Comment:
The file name `test_kokkos.py` is very broad and doesn't speak too much about what you are testing on it.

Reply:
Yes. I could rename it `test_kokkos_utils.py` or simply `test_kokkos_initargs.py` if that's all it is going to test.